API/getUserById/getUserById.py

```diff
@@ -0,0 +1,80 @@
+import boto3
+from botocore.exceptions import ClientError
+import logging
+import json
+import os 
+
+dynamodb = boto3.resource('dynamodb')
+
+
+def lambda_handler(event, context):
+    """ 
+	A Lambda function used by the APIs from 100daysofcloud.com to get user details by Id
+
+	Parameters: 
+	event["queryStringParameters"]["username"] (string): 100daysofcloud user name
+  
+	Returns: 
+	object: Returns user details from 100daysofcloud account if user is valid.
+  
+	"""
+    logging.basicConfig(
+        level=logging.INFO,
+        format=f'%(asctime)s %(levelname)s %(message)s'
+    )
+
+    logger = logging.getLogger()
+
+    git_user=""
+    resp={}
+    statusCode=404
+    
+    #checking if the lambda is called by an API gateway and has query params in the call
+    if "queryStringParameters" in event:
```

Comment:
I don't think it is necessary to check for this, as this won't be an user-facing API endpoint, so we already know where this will be used and called from.